### ISSUE TITLE
Fetch template at runtime

### ISSUE THREAD

**Issue by owenthewizard** [Kobzol/cargo-wizard#22]
I think it would be cool to add the (optional) ability to fetch the latest template at runtime, so that it could be updated without updating the binary.

**Comment by Kobzol** [comment]
Yeah, that could be useful, however it's relatively complex (introducing network requests into the CLI app, handling timeouts, offline mode, deciding how often to check, handling security of the remote side etc.), and the current templates haven't been changing often so far. Eventually, it would be nice if cargo-wizard had the ability to load templates from some config files on disk, and it could then download the newest files from GitHub directly.
